src/App.js

```diff
@@ -1,15 +1,26 @@
 import React from 'react';
+import { connect } from 'react-redux'
```

Comment:
1. `App.js` doesn't need to connect to the `redux` store so we can safely remove `connect` as well as `mapStateToProps`.

2. `App` component doesn't need to be a `class`